src/journal/lookup3.c

```diff
@@ -772,8 +772,13 @@ uint32_t jenkins_hashbig( const void *key, size_t length, uint32_t initval)
 
 #ifdef SELF_TEST
 
+#define hashlittle jenkins_hashlittle
+#define hashlittle2 jenkins_hashlittle2
+#define hashword jenkins_hashword
+#define hashword2 jenkins_hashword2
+
```

Comment:
I don't understand why this is necessary.

Reply:
> Please add this commit to your PR after it all compiles:
> 
> [0001-tests-turn-lookup3-into-a-test.patch.txt](https://github.com/systemd/systemd/files/4589581/0001-tests-turn-lookup3-into-a-test.patch.txt)
This was a patch @keszybz wanted